apim-operator/build/Dockerfile

```diff
@@ -10,7 +10,8 @@ COPY build/_output/bin/apim-operator ${OPERATOR}
 COPY build/microgwconf.mustache /usr/local/bin
 COPY build/controller_resources/Dockerfile /usr/local/bin
 COPY build/dockerSecretTemplate.mustache /usr/local/bin
-COPY build/controller_resources/dockerFile.mustache /usr/local/bin
+#COPY build/controller_resources/dockerFile.mustache /usr/local/bin
```

Comment:
If we are going with gotmpl, then let's remove this line. No need to add commented code.

Reply:
removed